### PR TITLE
Add task 'Using Debugging Endpoints'

### LINKED ISSUE #281: Add instructions for using control-plane debug endpoints
linkerd/linkerd2#2739 introduced a series of `/debug/pprof/` endpoints to each control-plane component.

Add some documentation, probably near the Modifying Proxy Log Level page, on usage. A good starting point is in the PR's description:
https://github.com/linkerd/linkerd2/pull/2739#issue-272904091

Relates to #279

### PR DESCRIPTION
**Pull request by alpeb**
Fixes #281

### CODE REVIEW

File: linkerd.io/content/2/tasks/using-debug-endpoints.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  parameter. After you get the profile file, use the go tool pprof command to
+  investigate the profile.
+- jhreadcreate: Stack traces that led to the creation of new OS threads
+- trace: A trace of execution of the current program. You can specify the
+  duration in the seconds GET parameter. After you get the trace file, use the
+  go tool trace command to investigate the trace.
+
+## Example Usage
+
+This data is served over the same port as the main container's liveness probe.
```
Comment (ihcsim) [inline linkerd.io/content/2/tasks/using-debug-endpoints.md]: fwiw, the name of the port is `admin-http`.

File: linkerd.io/content/2/tasks/using-debug-endpoints.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+To find this port, you can examine the pod's yaml, or for the identity pod for
+example, issue a command like so:
+
+```bash
+kubectl -n linkerd get po linkerd-identity-8ccbfc5d6-css2v -o=jsonpath='{.spec.containers[?(@.livenessProbe.httpGet.path=="/ping")].livenessProbe.httpGet.port}'
```
Comment (ihcsim) [inline linkerd.io/content/2/tasks/using-debug-endpoints.md]: Maybe just fetch the `admin-http` container port?

File: linkerd.io/content/2/tasks/using-debug-endpoints.md (hunk 1 above)
Comment (alpeb) [reply linkerd.io/content/2/tasks/using-debug-endpoints.md]: Apparently the proxy injector and the sp validator still need to name that port

Comment (alpeb) [reply linkerd.io/content/2/tasks/using-debug-endpoints.md]: I've created linkerd/linkerd2#2951 to track that :+1:

File: linkerd.io/content/2/tasks/using-debug-endpoints.md (hunk 2 above)
Comment (alpeb) [reply linkerd.io/content/2/tasks/using-debug-endpoints.md]: So once we get the `admin-http` label set across all the containers we can simplify with something like
```
k -n linkerd get po linkerd-identity-65cd84dc96-4dvrv -o=jsonpath='{.spec.containers[*].ports[?(@.name=="admin-http")].containerPort}
```

File: linkerd.io/content/2/tasks/using-debug-endpoints.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+- block: Stack traces that led to blocking on synchronization primitives
+- cmdline: The command line invocation of the current program
+- goroutine: Stack traces of all current goroutines
+- heap: A sampling of memory allocations of live objects. You can specify the gc
+  GET parameter to run GC before taking the heap sample.
+- mutex: Stack traces of holders of contended mutexes
+- profile: CPU profile. You can specify the duration in the seconds GET
+  parameter. After you get the profile file, use the go tool pprof command to
+  investigate the profile.
+- jhreadcreate: Stack traces that led to the creation of new OS threads
```
Comment (dadjeibaah) [inline linkerd.io/content/2/tasks/using-debug-endpoints.md]: s/jhreadcreate/threadcreate

Comment (alpeb) [reply linkerd.io/content/2/tasks/using-debug-endpoints.md]: Good catch. As for the warnings it appears that comes from other parts, so I'll leave that for a separate PR.

File: linkerd.io/content/2/tasks/using-debug-endpoints.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+This data is served over the `admin-http` port.
+To find this port, you can examine the pod's yaml, or for the identity pod for
+example, issue a command like so:
+
+```bash
+kubectl -n linkerd get po linkerd-identity-8ccbfc5d6-css2v -o=jsonpath='{.spec.containers[*].ports[?(@.name=="admin-http")].containerPort}'
+```
+
+Then port-forward that port to access it from outside the cluster (in this
```
Comment (ihcsim) [inline linkerd.io/content/2/tasks/using-debug-endpoints.md]: TIOLI: _Then use the `kubectl port-forward` command to access it from outside the cluster_

**Pull request merged**